source/create_my_testbed.rst

```diff
@@ -59,9 +59,10 @@ Finally, for all optical system, you can use generic functions like to creaet an
     # Focal plane intensity for this aberrations
     PSF = generic_os.todetector_intensity(entrance_EF = input_wavefront)
 
+``todetector_intensity``'s keyword "in_contrast" can be used to normalized the PSF in contrast or not. 
+``center_on_pixel`` can be used to center the PSF in the center or not (False by default).
 
-
-In the next section, we will present the existing ``OpticalSystem`` (``Pupil``,
+In the later sections, we will present the existing ``OpticalSystem`` (``Pupil``,
```

Comment:
--> "In the later sections, we will present the existing ``OpticalSystem`` *subclasses* (``Pupil``, [...]"